src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TodoItem } from '../TodoItem';
+
+export const TodoList = ({
+  items,
+  changeStatus,
+  removeTodo,
+  changeTodo,
+}) => (
+  <ul className="todo-list">
+    {items.map(item => (
+      <TodoItem
+        key={item.id}
+        {...item}
+        changeStatus={changeStatus}
+        removeTodo={removeTodo}
+        changeTodo={changeTodo}
+      />
```

Comment:
if you pass item as the full object without spread operator, it will be clearer when you receive props

it's better to have props with methods and an item, which has its fields
current interface: you mix external methods with parts of the item object

Reply:
fixed